### PR TITLE
remove attr and modify url for token

### PR DESCRIPTION
**Pull request by hedera07**

### CODE REVIEW

File: ethereum/Webaverse.sol [hunk 1]
```diff
@@ -117,18 +116,16 @@ contract Webaverse is WebaverseVoucher, OwnableUpgradeable {
...
 
     /**
      * @notice Claims(Mints) the a single NFT with given parameters.
      * @param to The address on which the NFT will be minted(claimed).
-     * @param data The data to store when claim.
      * @param voucher A signed NFTVoucher that describes the NFT to be redeemed.
      **/
     function claim_NFT(
         address to,
-        bytes memory data,
```
Comment (jakezira) [inline ethereum/Webaverse.sol:131]: Don't we need data here?

File: ethereum/WebaverseERC1155.sol [hunk 2]
```diff
@@ -91,16 +91,16 @@ contract WebaverseERC1155 is
 
     /**
      * @return Returns the token URI against a particular token id.
-     * e.g. [baseURI]/[_id]
+     * e.g. It's just the token's content URL, which will most likely be a metaversefile directory. But it could also be a GH pages or similar.
```
Comment (jakezira) [inline ethereum/WebaverseERC1155.sol]: `contenturl` and `uri()` seems different.

Comment (jakezira) [reply ethereum/WebaverseERC1155.sol]: From avaer:
> there is the URL we set during mint which in our case will probably point to web3.storage
> that is not the same as the URL the contract returns for the API
> our API looks at the contract's url to download the metaversefile and serve the details
> but there are two entirely different URLS

**Review by jakezira — COMMENTED** [review]
Need more discussion

File: ethereum/Webaverse.sol (hunk 1 above)
Comment (hedera07) [reply ethereum/Webaverse.sol:131]: Yes. We don't need data in the claim.
Claiming is the transfer of tokens from the signer to the claimer.

File: ethereum/WebaverseERC1155.sol (hunk 2 above)
Comment (hedera07) [reply ethereum/WebaverseERC1155.sol]: So should uri() return URL like "https://tokens.webaverse.com/1"?

Comment (jakezira) [reply ethereum/WebaverseERC1155.sol]: Yes

File: ethereum/WebaverseERC1155.sol [hunk 3]
```diff
@@ -91,34 +91,36 @@ contract WebaverseERC1155 is
...
+    function setTokenContentURL(uint256 tokenId, string memory _uri)
         public
         onlyMinter
     {
         require(bytes(_uri).length > 0, "ERC1155: URI must not be empty");
         _tokenURIs[tokenId] = _uri;
     }
 
+    /**
+     * @dev get token ContentURL
```
Comment (jakezira) [inline ethereum/WebaverseERC1155.sol]: `contentUrl` as it's variable named, not with first capital

File: ethereum/WebaverseERC1155.sol [hunk 4]
```diff
@@ -112,12 +112,12 @@ contract WebaverseERC1155 is
...
     /**
-     * @dev get token ContentURL
-     * @param tokenId Token id to get the ContentURL
+     * @dev get token contentURL
+     * @param tokenId Token id to get the contentURL
      */
     function getTokenContentURL(uint256 tokenId) public view returns (string memory) 
     {
-        require(currentTokenId >= tokenId, "ERC1155: ContentURI query for nonexistent token");
+        require(currentTokenId >= tokenId, "ERC1155: contentURI query for nonexistent token");
```
Comment (jakezira) [inline ethereum/WebaverseERC1155.sol]: `contentURL` not `contentURI`

File: ethereum/WebaverseERC1155.sol (hunk 3 above)
Comment (hedera07) [reply ethereum/WebaverseERC1155.sol]: fixed it

File: ethereum/WebaverseERC1155.sol (hunk 4 above)
Comment (hedera07) [reply ethereum/WebaverseERC1155.sol]: fixed it

**Pull request merged**